pkg/utils/utils_test.go

```diff
@@ -0,0 +1,262 @@
+package utils_test
+
+import (
+	multiv1beta1 "github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
+	netdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net"
+	"os"
+
+	"github.com/Mellanox/multi-networkpolicy-tc/pkg/utils"
+)
+
+var _ = Describe("utils test", func() {
+	Context("CheckNodeNameIdentical()", func() {
+		It("check node name identical with domain", func() {
+			h1 := "my-host.some-lab.some-site"
+			h2 := "my-host.some-lab-alt-name.some-site-alt-name"
+			Expect(utils.CheckNodeNameIdentical(h1, h2)).To(BeTrue())
+		})
+		It("check node name identical with domain on first arg", func() {
+			h1 := "my-host.some-lab.some-site"
+			h2 := "my-host"
+			Expect(utils.CheckNodeNameIdentical(h1, h2)).To(BeTrue())
+		})
+		It("check node name identical with domain on second arg", func() {
```

Comment:
It looks like this test is identical with the previous one

Reply:
good catch, meant to swap the call with (h2, h1) but i ended up now with changing values as it reads better